presets/airbnb.json

```diff
@@ -24,7 +24,7 @@
     ],
     "disallowMixedSpacesAndTabs": true,
     "disallowTrailingWhitespace": true,
-    "disallowTrailingComma": true,
+    "disallowTrailingComma": false,
```

Comment:
I put ignoreSingleLine, because the argument about the trailing comma is for pull requests, and not affected by a single line object / array